raw_tracepoints/syscalls.go

```diff
@@ -0,0 +1,21 @@
+package raw_tracepoints
+
+import (
+	"unsafe"
+
+	"github.com/cilium/ebpf/internal"
+)
+
+const (
+	rawTracepointOpen = 17
```

Comment:
To avoid import loops, we can not use `_RawTracepointOpen` from [1]. But keeping the same name schema would be great.

[1] https://github.com/cilium/ebpf/blob/master/types.go#L112